webapp/tests/data_access/test_form_data_controller.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from unittest.mock import patch
+
+import fakeredis as fakeredis
```

Comment:
Why do we use redislite and fakreredis? Can we use redislite.StrictRedis() from the form_data_controller?

Reply:
I thought for the tests we should not be using the actual implementation of the controller, specially regarding the redis connection. Furthermore using a mocking fake-redis we can do teardown deletion of the database after each test

Reply:
That was my idea jep, let's see what Sanny thinks?